pkgs/tools/networking/hp2p/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, openmpi, python2Packages, autoconf, automake } :
+
+stdenv.mkDerivation rec {
+  pname = "hp2p";
+  version = "3.3";
+
+  src = fetchFromGitHub {
+    owner = "cea-hpc";
+    repo = "hp2p";
+    rev = "${version}";
+    sha256 = "0zvlwb941rlp3vrf9yzv7njgpj3mh4671ch7qvxfa4hq2ivd52br";
+  };
+
+  enableParallelBuilding = true;
+  nativeBuildInputs = [ autoconf automake python2Packages.wrapPython ];
+  buildInputs = [ openmpi ] ++ (with python2Packages; [ python numpy matplotlib plotly ]) ;
+  pythonPath = (with python2Packages; [ numpy matplotlib plotly ]) ;
+
+  preConfigure = ''
+    patchShebangs .
+    ./autogen.sh
+    export CC=mpicc
+    export CXX=mpic++
```

Comment:
I am not sure if that works cross compile.

Reply:
We probably do not need to worry about it. None of the MPI stuff is built with cross compilation in mind. It would probably require to make all MPI implementations cross compilation friendly first.